Services/UserService/Commands/CreateUserCommand.cs

```diff
@@ -1,10 +1,10 @@
 ﻿using DTO;
-using UserService.Interfaces;
 using Kernel;
+using System;
 
 namespace UserService.Commands
 {
-    public class CreateUserCommand : ICreateUser<string, string>
```

Comment:
why you remove interface, we need them if we want create good unit tests

Reply:
Why ? What interfaces can make to improve unit tests? 
I thought it was useless, if I user only ICreateCommand

Reply:
fixed